backend/controller/admin.controller.js

```
@@ -18,8 +19,10 @@ async function createAdmin(req, res) {
     return res.status(400).json({ error: validation.error.errors });
   }
   const existingAdmin = await Admin.findOne({ email: req.body.email });
+  const existingAdmin = await Admin.findOne({ email: req.body.email });
   if (existingAdmin) {
     return res.status(409).json({ error: "Email is already registered" });
+    return res.status(409).json({ error: "Email is already registered" });
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)